Makefile

```diff
@@ -597,29 +597,27 @@ html-docs-dockerized: build-test ## Build html docs dockerized
 		make html-docs
 
 .PHONY: fmt
-fmt: ## Format the code (using black and isort)
+fmt: ## Format the code using black
```

Comment:
make fmt should also format the imports. maybe need to add ruff here as well

Reply:
👆

Reply:
Ok, I've added it to this target: 299314d.
The direction here is to stay with a single command that lints, formats, and checks - it fixes what it can and reports the overall status 🙂